client/src/containers/AddPostContainer/AddPostContainer.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { AddPostContainer } from './index';
+
+import { shallow } from 'enzyme';
+import configureStore from 'redux-mock-store';
+import React from 'react';
+import { Provider } from 'react-redux';
+
+describe('AddPostContainer component:', () => {
+    const props = {
+        croppedImage: 'someimg',
+        description: 'somedescription',
+        loading: false,
+        uploadPost: jest.fn(() => 'somevalue'),
+        setCroppedImageForPost: jest.fn(() => 'somevalue'),
+        setDescriptionForPost: jest.fn(() => 'somevalue'),
+        resetAddPost: jest.fn(() => 'somevalue'),
+        informFileError: jest.fn(() => 'somevalue'),
+        loggedUsername: jest.fn(() => 'somevalue'),
+    };
+    let renderer;
```

Comment:
why let? when do u change it?

Reply:
![Screenshot_11](https://user-images.githubusercontent.com/49571036/66269452-37f04900-e851-11e9-9ca5-c1f2fa09b56b.png)